frontend/src/app.vue

```diff
@@ -36,7 +36,7 @@ const loadingResourcesMessage = 'Loading resources...';
 const app = defineComponent({
   name: 'app',
   components: {
-    LoadingOverlay,
+    Loading,
```

Comment:
Can this still be named `LoadingOverlay`? I think that would make it more meaningful.